### DISCUSSION TITLE
Epoch 1.16 hangs

### DISCUSSION THREAD

**Discussion by leeyemi** [tearust/teaproject#17] [General]
Hi,

The wallet hangs when accessing logs (around block heights 3300) it was not hanged before.
Experienced this one as well when testing epoch 1.15 (block height over 100k)

Thanks,
Yemi

**Comment by liyangwood** [comment]
Thanks for issue.
We updated the log service just now to make it more stable, hope the log service not hangs again.

**Comment by kevingzhang** [comment]
If it has been fixed, please mark solved.
